src/main/java/fi/hsl/common/transitdata/TransitdataProperties.java

```diff
@@ -103,4 +105,8 @@ public static long currentTimestamp() {
     public static String formatJoreId(String route, String direction, String startDate, String startTime) {
         return REDIS_PREFIX_JORE_ID + route + "-" + direction + "-" + startDate + "-" + startTime;
     }
+
+    public static String formatJoreId(String route, String direction, String startDate, ThirtyHourTime startTime) {
```

Comment:
Should we have another class which bundles startDate (or actually operatingDate) and ThirtyHourTime together. Those should probably always go hand in hand..? Not sure, perhaps ask from @laaksma

Reply:
I was thinking this would be just an explicit and reusable way to do the time conversion (rather than a wrapper for datetime), but if they are always bundled that might be ok.

Reply:
Added date to the class.

Reply:
Not sure about this. I though of having a second class that composes date and ThirtyHourTime. But you should talk to @laaksma about this, I'm not the expert here.

Reply:
@laaksma mentioned a use case in which we may want to compare two time points so combining makes sense.